public/app.js

```diff
@@ -0,0 +1,395 @@
+const APIController = (function() {
+
+    const clientId = 'e91988fcc1674aa88374f24be4844f85';
+    const clientSecret = '10badbb697f544ecb15e1efd7659a0b0';
+
+
+    /**
+    * @description Функция получения Токена Spotify
+    */
+    const _getToken = async () => {
+
+        const result = await fetch('https://accounts.spotify.com/api/token', {
+            method: 'POST',
+            headers: {
+                'Content-Type' : 'application/x-www-form-urlencoded',
+                'Authorization' : 'Basic ' + btoa(clientId + ':' + clientSecret)
+            },
+            body: 'grant_type=client_credentials'
+        }).catch(err => alert(err));
+
+        const data = await result.json();
+        return data.access_token;
+    }
+    /**
+    * @description Функция получения жанров
+    * @param {string} token - токен Spotify
+    */
+    const _getGenres = async (token) => {
+
+        const result = await fetch(`https://api.spotify.com/v1/browse/categories?locale=sv_RU`, {
+            method: 'GET',
+            headers: { 'Authorization' : 'Bearer ' + token}
+        }).catch(err => alert(err));
```

Comment:
err это объект
тут в alert сейчас выведется `[Object Object]`
Должно быть:
```suggestion
        }).catch(err => alert(err.message));
```